rama-http-types/src/curl/headers.rs

```diff
@@ -0,0 +1,13 @@
+use rama_core::error::OpaqueError;
+
+pub fn request_headers_to_curl_command<T: IntoCurlHeadersCommand>(request: T) -> Result<String, OpaqueError> {
```

Comment:
FYI in case you have an `AcceptEncoding` header also do not forget to add the `--compressed` flag at the end